src/rust/catloop/futures/accept.rs

```diff
@@ -47,7 +47,7 @@ use ::std::{
 enum ServerState {
     ListenAndAccept(QToken),
     Connect(QToken, Rc<DuplexPipe>),
-    Connected(QDesc, SocketAddrV4, Rc<DuplexPipe>),
+    Connected(QToken, QDesc, SocketAddrV4, Rc<DuplexPipe>),
```

Comment:
We might consider using named fields here since there are four of them and it might be hard to remember the order they should be listed later.

Reply:
Sure!

Reply:
Sure. I applied the change. let me know if you prefer it this way.